.travis.yml

```diff
@@ -3,10 +3,19 @@ language: python
 matrix:
   include:
   - python: '2.7'
+    env:
+      IPYTHON_VERSION='5.8'
   - python: '3.5'
+    env:
+      IPYTHON_VERSION='7.9'
```

Comment:
Is this because ipython > 7.9 doesn't support python3.5?

Reply:
That's right. 7.9 was the last release to support python 3.5, same logic for the choice of version for python 2.7